src/status_im/contexts/wallet/account/view.cljs

```diff
@@ -28,11 +28,14 @@
       (let [{:keys [name color formatted-balance
                     watch-only?]} (rf/sub [:wallet/current-viewing-account])
             customization-color   (rf/sub [:profile/customization-color])]
-        (rn/use-unmount #(rf/dispatch [:wallet/close-account-page]))
+        (rn/use-unmount (fn []
+                          (rf/dispatch [:wallet/close-account-page])
+                          (rf/dispatch [:wallet/clean-send-data])))
         [rn/view {:style {:flex 1}}
          [account-switcher/view
           {:type     :wallet-networks
-           :on-press #(rf/dispatch [:wallet/close-account-page])}]
+           :on-press (fn []
+                       (rf/dispatch [:wallet/close-account-page]))}]
```

Comment:
This would be `#(rf/dispatch navigate-back)` as now we are already dispatching `wallet/close-account-page` on unmount

Reply:
I think  we don't need `navigate-back` and  `close-account-page` together since `close-account-page` already goes to the root of the navigation.

So I'll keep the `on-press` as is, and the use-effect will look as:

`(rn/use-unmount #(rf/dispatch [:wallet/clean-send-data]))`

Reply:
I'll rever the suggested changes because of:
- https://github.com/status-im/status-mobile/pull/20183#discussion_r1619394309